### PR TITLE
Remove app/labor/color_from_image.rb

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Refactor

## Description

This PR removes the `ColorFromImage` class from `app/labor`, as well as the `Articles::UpdateMainImageBackgroundHexWorker` which only existed to call this object. 

Here's some context from @benhalpern:

> [Disabling the Miro gem] was originally the PR which put us over the Heroku threshold for slug size and this gem was pulled in order to make that individual deploy succeed.
>
> However, the Miro gem is pretty old and has a dependency which was renamed and the code is in master, but the latest release doesn't reflect that. So there was a question of whether it was a good dependency for the app to have.
>
> This functionality was put in place to replace an old library which handled this behavior which also had gone unsupported, so it's been a bit of a journey. I like the automatic background color detection, but could see a case for just removing it. We don't do this on other images besides the main image and we clearly have lived with this non-functional endpoint for long enough.
>
>  If we really want this again in the future we can find a cleaner overall solution.

## Related Tickets & Documents

https://github.com/forem/InternalProjectPlanning/issues/271

## QA Instructions, Screenshots, Recordings

Nothing specific

- [X] No, and this is why: only removing code, not adding any new one

## Added to documentation?

- [X] No documentation needed

### CODE REVIEW

**Review by rhymes — APPROVED** [review]
🪓

**Comment by rhymes** [comment]
@citizen428 forgot to mention in the review, not blocking obviously, we might want to get rid of this line too: https://github.com/forem/forem/blob/66dc8081b11525826885311c5eebd385f6e1eff0/Gemfile#L60

**Comment by citizen428** [comment]
@rhymes Thanks, done ✅

File: TODO [hunk 1]
```diff
@@ -0,0 +1 @@
+* look into profile updates to remove blank values
```
Comment (maestromac) [inline TODO:1]: :eyes: did you accidentally commit this @citizen428 ?

**Review by Ridhwana — APPROVED** [review]

**Review by benhalpern — APPROVED** [review]

File: TODO (hunk 1 above)
Comment (citizen428) [reply TODO:1]: Yes, I did 🤦 It's my private TODO list...

**Pull request merged**